macros/path_transformations/transform_paths.sql

```diff
@@ -6,31 +6,31 @@
 
 {% macro default__transform_paths(model_type, source_cte) %}
 
-  {% set allowed_path_transforms = ['exposure_path', 'first_path', 'frequency_path', 'remove_if_last_and_not_all', 'remove_if_not_all', 'unique_path'] %}
+  {% set allowed_snowplow__path_transforms = ['exposure_path', 'first_path', 'frequency_path', 'remove_if_last_and_not_all', 'remove_if_not_all', 'unique_path'] %}
 
-  , path_transforms as (
+  , snowplow__path_transforms as (
```

Comment:
I think you might have caught a few things with a find and replace you maybe weren't meaning to? In a few other places as well, will try and tag them.

Reply:
Good catch, it is all due to the path_transform variable. Will sort all.